apps/api/src/app/workflows/notification-template.controller.ts

```diff
@@ -191,6 +191,7 @@ export class NotificationTemplateController {
         data: body.data,
         __source: query?.__source,
         type: WorkflowTypeEnum.REGULAR,
+        origin: WorkflowOriginEnum.NOVU,
```

Comment:
remove from here

Reply:
@djabarovgeorge should not be removed here, it's using the inner commands of create workflow